docs/streams/upgrade-guide.html

```diff
@@ -34,71 +34,45 @@ <h1>Upgrade Guide and API Changes</h1>
-    </p>
-
-    <p>
-        Upgrading from 0.10.0.x to 2.0.0 directly is also possible.
-        Note, that a brokers must be on version 0.10.1 or higher and on-disk message format must be on version 0.10 or higher
-        to run a Kafka Streams application version 2.0 or higher.
-        See <a href="#streams_api_changes_0101">Streams API changes in 0.10.1</a>, <a href="#streams_api_changes_0102">Streams API changes in 0.10.2</a>,
-        <a href="#streams_api_changes_0110">Streams API changes in 0.11.0</a>, <a href="#streams_api_changes_100">Streams API changes in 1.0</a>, and
-        <a href="#streams_api_changes_110">Streams API changes in 1.1</a>, and <a href="#streams_api_changes_200">Streams API changes in 2.0</a>
-        for a complete list of API changes.
-        Upgrading to 2.0.0 requires two rolling bounces with config <code>upgrade.from="0.10.0"</code> set for first upgrade phase
-        (cf. <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-268%3A+Simplify+Kafka+Streams+Rebalance+Metadata+Upgrade">KIP-268</a>).
-        As an alternative, an offline upgrade is also possible.
+        There are a few new APIs added and also some deprecated APIs since 1.0.x is removed in 2.0.0, and hence users upgrading may need to make some small code changes (see below for the details).
+        In addition, upgrading from any older version to 2.0.0 in the online mode requires two rolling bounces.
+        In the first rolling bounce phase users need to set config <code>upgrade.from="0.10.0"</code>
+        (cf. <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-268%3A+Simplify+Kafka+Streams+Rebalance+Metadata+Upgrade">KIP-268</a>):
     </p>
     <ul>
         <li> prepare your application instances for a rolling bounce and make sure that config <code>upgrade.from</code> is set to <code>"0.10.0"</code> for new version 2.0.0</li>
         <li> bounce each instance of your application once </li>
         <li> prepare your newly deployed 2.0.0 application instances for a second round of rolling bounces; make sure to remove the value for config <code>upgrade.mode</code> </li>
         <li> bounce each instance of your application once more to complete the upgrade </li>
     </ul>
-    <p> Upgrading from 0.10.0.x to 2.0.0 in offline mode: </p>
+    <p> As an alternative, an offline upgrade is also possible. Upgrading from 0.10.0.x to 2.0.0 in offline mode: </p>
     <ul>
         <li> stop all old (0.10.0.x) application instances </li>
         <li> update your code and swap old code and jar file with new code and new jar file </li>
         <li> restart all new (2.0.0) application instances </li>
     </ul>
 
+    <p>
+        Note, that a brokers must be on version 0.10.1 or higher and on-disk message format must be on version 0.10 or higher
+        to run a Kafka Streams application version 0.10.1 or higher.
```

Comment:
Not completely correct. On-disk message format 0.10 required since Streams 1.0 -- older Streams version work with older message format.